dll/win32/msgina/shutdown.c

```diff
@@ -15,6 +15,7 @@
 #define WLX_SHUTDOWN_STATE_LOGOFF       0x01
 #define WLX_SHUTDOWN_STATE_POWER_OFF    0x02
 #define WLX_SHUTDOWN_STATE_REBOOT       0x04
+#define WLX_SHUTDOWN_STATE_NT_MODE      0x05
```

Comment:
This does not seem correct, since it is used as bitflag (See `return WLX_SHUTDOWN_STATE_POWER_OFF | WLX_SHUTDOWN_STATE_REBOOT | WLX_SHUTDOWN_STATE_NT_MODE;`)